adafruit_pcf8591/pcf8591.py

```diff
@@ -85,7 +97,7 @@ def _half_read(self, channel):
         with self.i2c_device as i2c:
             i2c.write_then_readinto(self._buffer, self._buffer)
 
-    def read(self, channel):
+    def read(self, channel: Literal[0, 1, 2, 3]) -> None:
```

Comment:
Typing is correct, but the docstring parameters are broken here as well, if you wouldn't mind fixing it and adding the type to it as well.